src/DynamoUtilities/DynamoPathManager.cs

```diff
@@ -148,7 +148,12 @@ public void InitializeCore(string mainExecPath)
             CommonSamples = Path.Combine(commonData, "samples", UICulture);
```

Comment:
Should we set the UICulture en-US, if the language pack for the current culture is not present?